examples/types-use-ipfs-from-typed-js/src/main.js

```diff
@@ -1,4 +1,8 @@
-const IPFS = require('ipfs')
+const { create } = require('ipfs')
+/**
+ * @typedef {import('ipfs').IPFS} IPFS 
+ * @typedef {import('cids')} CID
+ */
 
 async function main () {
   const node = await IPFS.create()
```

Comment:
Is IPFS defined now?  The require was deleted.

Reply:
Ah, I think the test for this module isn't being run in CI because it's name in the `package.json` does not start with `example-` - see https://github.com/ipfs/js-ipfs/blob/master/.travis.yml#L243

Reply:
I've updated the PR so that the tests now run in CI